src/block/block_ext.c

```diff
@@ -1174,9 +1174,9 @@ __wt_block_extlist_read(WT_SESSION_IMPL *session,
 	WT_DECL_ITEM(tmp);
 	WT_DECL_RET;
 	wt_off_t off, size;
+	const uint8_t *p;
```

Comment:
This was unexpected, I would have thought `uint8_t` would sort after `int`, but if you think it's right, fbm.

Reply:
Yeah it doesn't make much sense to me but that's the order specified in `function.py`.
The reason why I've made this change is because the new line length means that some multiline declarations (which previously caused `function.py` to stop parsing) have been collapsed into a single line. So after formatting, we're able to detect these ordering issues for the first time.

Reply:
That one (in retrospect), continues to feel right -- "int" is an undefined size, so shrug.